src/plugins/sound_gstreamer.py

```diff
@@ -1,7 +1,5 @@
 # -*- coding: utf-8 -*-
 """
-src/plugins/sound_gstreamer.py
```

Comment:
maybe just `Sound plugin` too, it probably just uses a different backend

Reply:
Sound notification plugin using gstreamer